docs/apm/apm-alerts.asciidoc

```diff
@@ -11,44 +11,52 @@ and trigger built-in **actions** when those conditions are met.
 
 The following **rules** are supported:
 
-* Latency anomaly rule:
-Alert when latency of a service is abnormal
-* Transaction error rate threshold rule:
-Alert when the service's transaction error rate is above the defined threshold
-* Error count threshold rule:
-Alert when the number of errors in a service exceeds a defined threshold
+* **Threshold rule**:
+Alert when the latency or failed transaction rate is abnormal.
+Threshold rules can be as broad or as granular as you'd like, enabling you to define exactly when you want to be alerted--whether that's at the environment level, service name level, transaction type level, and/or transaction name level.
+* **Anomaly rule**:
+Alert when either the latency, throughput, or failed transaction rate of a service is anomalous. Anomaly rules can be set at the environment level, service level, and/or transaction type level.
```

Comment:
I think alerts can only be created for latency anomalies (https://github.com/elastic/kibana/issues/159288)

Reply:
Aha! Thanks.
```suggestion
Alert when either the latency of a service is anomalous. Anomaly rules can be set at the environment level, service level, and/or transaction type level.
```